pkgs/applications/editors/uivonim/default.nix

```diff
@@ -0,0 +1,73 @@
+{ lib, mkYarnPackage, fetchFromGitHub, electron, makeWrapper }:
+
+mkYarnPackage rec {
+  pname = "uivonim";
+  version = "20210524";
```

Comment:
```suggestion
  version = "unstable-2021-05-24";
```

Reply:
Hopefully we can make some progress on https://discourse.nixos.org/t/brainstorming-for-rfc-pname-and-version/12873/36 soon.  This "unstable-" prefix is super inconvenient because it breaks nix-env.

I've changed it anyhow because the prefix is still technically recommended by the manual.

Reply:
for a big package such as  uiveonim (e.g., more likely to  be installed via nix-env), and also if that fits the maintainer workflow better, I think the manual could be overruled. I haven't followed closely but the `unstable` thing has been undecisive for a long time. We can change it once a consensus has been reached ?

Reply:
It's unfortunately not that clear-cut.  Using `2021-05-24` without the prefix as version breaks *another* feature of nix-env, namely `nix-env -u` (because it won't subsequently upgrade from 2021-05-24 to 0.29.0).  IMHO we should just default to the `--always` flag here, and then this wouldn't be an issue.

I agree that we should wait for the discussion to conclude, and then change it afterwards.

It's just a bit frustrating that the original "unstable-" convention was added without much thought or discussion, and now end up staying in the eternal bikeshedding inferno.